llama_index/tools/types.py

```diff
@@ -42,7 +42,10 @@ def to_openai_function(self) -> Dict[str, Any]:
         The name and arguments of a function that should be called, as generated by the
         model.
         """
-        # TODO fix this and make sure nothing breaks
+        return self.to_openai_tool()
```

Comment:
@jerryjliu [Comment on this from original PR](https://github.com/run-llama/llama_index/pull/8738#discussion_r1386000716)
> is this just duplicate code with to_openai_function? Can we just have to_openai_function call to_openai_tool while still marking it as deprecated?